data.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Holds the data associated with a website. Websites are identified by origin.
+ */
+class AkitaOriginData {
+  origin = null;
+  isMonetized = false;
+  // Object<AkitaPaymentPointerData>
```

Comment:
is this needed?

Reply:
No, just annotating the type to try to make it more clear how the data is structured. Modified the comment so it is hopefully more clear what it means.